### PR TITLE
fix: only export the default export's props

### PR DESCRIPTION
**Pull request by hkc452**
##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [x] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

**Comment by sorrycc** [comment]
ci 挂了，用例没过，修下。

File: packages/ast/src/getExportProps/getExportProps.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ function findAssignmentExpressionProps(opts: {
       t.isAssignmentExpression(node) &&
       t.isMemberExpression(node.left) &&
       t.isIdentifier(node.left.object) &&
+      node.left.object.name === opts.name &&
```
Comment (ycjcl868) [inline packages/ast/src/getExportProps/getExportProps.ts]:
```suggestion
      node.left.object?.name === opts.name &&
```

Comment (sorrycc) [reply packages/ast/src/getExportProps/getExportProps.ts]: 前面有判断了。

File: packages/ast/src/getExportProps/getExportProps.ts [hunk 2]
```diff
@@ -46,6 +46,7 @@ function findAssignmentExpressionProps(opts: {
       t.isAssignmentExpression(node) &&
       t.isMemberExpression(node.left) &&
       t.isIdentifier(node.left.object) &&
+      node.left.object?.name === opts.name &&
```
Comment (sorrycc) [inline packages/ast/src/getExportProps/getExportProps.ts]:
```suggestion
      node.left.object.name === opts.name &&
```

**Pull request merged**